protos/sources/data-node/grpc-rest-bindings.yml

```diff
@@ -217,7 +217,7 @@ http:
     - selector: datanode.api.v2.TradingDataService.ListTrades
       get: "/api/v2/trades"
     - selector: datanode.api.v2.TradingDataService.GetMarket
-      get: "/api/v2/market/{market_id}"
+      get: "/api/v2/market/{id}"
```

Comment:
ditto

Reply:
Same as reasoning here https://github.com/vegaprotocol/vega/pull/5895#discussion_r948798892 - made the market query consistent